pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -1,5 +1,7 @@
-{ buildFHSUserEnv, makeDesktopItem, writeScript, stdenv, lib, requireFile, unstick,
-  supportedDevices ? [ "Arria II" "Cyclone V" "Cyclone IV" "Cyclone 10 LP" "MAX II/V" "MAX 10 FPGA" ] }:
+{ buildFHSUserEnv, makeDesktopItem, writeScript, stdenv, lib, requireFile, unstick
+, supportedDevices ? [ "Arria II" "Cyclone V" "Cyclone IV" "Cyclone 10 LP" "MAX II/V" "MAX 10 FPGA" ]
+, postInstall ? ""
```

Comment:
I would prefer some more structured options here. For example, your suggested change could have:

- `disableCCompiler = true`
- `platforms = [ "twentynm" ]`

I don't know which parts of Quartus can be removed in this way, so I'll leave it to you to design a good interface here.